packages/ui-components/src/NavLink.tsx

```diff
@@ -4,22 +4,28 @@
 
 import React from 'react';
 
-import { StyledNavLink } from './Shared.styles';
+import { DynamicSizeText, StyledNavLink } from './Shared.styles';
 
 type Props = {
   children?: React.ReactNode,
-  value?: string,
-  to: string
+  fontSize?: string,
+  fontWeight?: string,
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void,
+  to: string,
+  value?: string
 };
 
 export default class NavLink extends React.PureComponent<Props> {
   render () {
-    const { children, to, value } = this.props;
+    const { children, fontSize = '17px', fontWeight = '300', onClick, to, value } = this.props;
 
     return (
-      <StyledNavLink to={to}>
-        {value}
-        {children}
+      <StyledNavLink onClick={onClick} to={to}>
```

Comment:
Small discussion (definitely for a future PR, if accepted): I would have renamed this component to `SecondaryButton`. My reasons:
- not everyone uses `react-router`, ui-components shouldn't import `react-router` to cater for more people
- the `<Link>` could be around anything: a span, div, PrimaryButton, SecondaryButton, and the end-developer takes care of that.

Reply:
+1